lxd/instances_get.go

```diff
@@ -98,6 +99,55 @@ func urlInstanceTypeDetect(r *http.Request) (instancetype.Type, error) {
 //   "500":
 //     $ref: "#/responses/InternalServerError"
 
+// swagger:operation GET /1.0/instances?all-projects=true instances instances_get_all_projects
```

Comment:
Instead of adding 3 new routes to swagger, I think it'd have been fine to just add a new `parameter` entry to the existing 3 routes.

Reply:
I may have caused this confusion though as initially I was thinking of using a different recursion level for this, which would have needed a full entry, but with all-projects=true, we don't need to, it's just an extra parameter.